Index: OutlookPlugin.rc
===================================================================
--- OutlookPlugin.rc	(revision 109)
+++ OutlookPlugin.rc	(working copy)
@@ -48,27 +48,27 @@
 STRINGTABLE 
 BEGIN
     IDR_MAINFRAME           "OutlookPlugin\n\\OutlookPlugin\n\n\\OutlookPlugin.Document\\OutlookPlugin Document"
-    IDS_SYNCTYPE1           "Two-way"
-    IDS_SYNCTYPE2           "One-way: Server -> Outlook"
-    IDS_SYNCTYPE3           "One-way: Outlook -> Server"
-    IDS_ACCOUNT_SETTINGS    "Account Settings"
-    IDS_ACCOUNT_URL         "&Location:"
-    IDS_ACCOUNT_USERNAME    "&Username:"
-    IDS_ACCOUNT_PASSWORD    "Pa&ssword:"
-    IDS_ACCOUNT_USEPROXY    "Enable"
-    IDS_ACCOUNT_PROXY_AUTH  "Requires authorization"
-    IDS_ACCOUNT_PROXY_USER  "&Username:"
-    IDS_ACCOUNT_PROXY_PASS  "Password:"
+    IDS_SYNCTYPE1           "Zwei-Weg"
+    IDS_SYNCTYPE2           "Ein-Weg: Server -> Outlook"
+    IDS_SYNCTYPE3           "Ein-Weg: Outlook -> Server"
+    IDS_ACCOUNT_SETTINGS    "Kontoeinstellungen"
+    IDS_ACCOUNT_URL         "&URL:"
+    IDS_ACCOUNT_USERNAME    "&Benutzername:"
+    IDS_ACCOUNT_PASSWORD    "Pa&sswort:"
+    IDS_ACCOUNT_USEPROXY    "Aktivieren"
+    IDS_ACCOUNT_PROXY_AUTH  "Benötigt Autorisierung"
+    IDS_ACCOUNT_PROXY_USER  "&Benutzername:"
+    IDS_ACCOUNT_PROXY_PASS  "Passwort:"
     IDS_OK                  "&OK"
-    IDS_CANCEL              "&Cancel"
-    IDS_ACCOUNT             "Account"
-    IDS_OUTLOOK_FOLDER      "Outlook Folder:"
+    IDS_CANCEL              "&Abbrechen"
+    IDS_ACCOUNT             "Benutzerkonto"
+    IDS_OUTLOOK_FOLDER      "Zielordner auswählen:"
 END
 
 STRINGTABLE 
 BEGIN
     AFX_IDS_APP_TITLE       "Funambol"
-    AFX_IDS_IDLEMESSAGE     "Ready"
+    AFX_IDS_IDLEMESSAGE     "Bereit"
 END
 
 STRINGTABLE 
@@ -83,220 +83,220 @@
 
 STRINGTABLE 
 BEGIN
-    ID_APP_ABOUT            "Display program information, version number and copyright\nAbout"
-    ID_APP_EXIT             "Quit the application\nExit"
+    ID_APP_ABOUT            "Programminformationen anzeigen, Version und Urheberrecht\nÜber"
+    ID_APP_EXIT             "Die Anwendung beenden\nBeenden"
 END
 
 STRINGTABLE 
 BEGIN
-    ID_VIEW_GUIDE           "Open Funambol Outlook Sync Client User Guide"
+    ID_VIEW_GUIDE           "Benutzerhandbuch (in Englisch) öffnen"
 END
 
 STRINGTABLE 
 BEGIN
-    ID_FILE_CONFIGURATION   "Configuration Options"
-    ID_FILE_SYNCHRONIZE     "Perform the sync"
-    ID_TOOLS_FULLSYNC       "Open full sync window"
-    ID_TOOLS_SETLOGLEVEL    "Set log level"
+    ID_FILE_CONFIGURATION   "Konfigurationsoptionen"
+    ID_FILE_SYNCHRONIZE     "Daten synchronisieren"
+    ID_TOOLS_FULLSYNC       "Das ganze Synchronisationsfester öffnen"
+    ID_TOOLS_SETLOGLEVEL    "Das Log-Level einstellen"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_INCLUDE_SUBFOLDERS  "I&nclude sub-folders"
-    IDS_SELECT_FOLDER       "C&hange..."
-    IDS_REMOTE_NAME         "R&emote name:"
+    IDS_INCLUDE_SUBFOLDERS  "U&nterordner miteinbeziehen"
+    IDS_SELECT_FOLDER       "Ä&ndern..."
+    IDS_REMOTE_NAME         "R&emote-Name:"
     IDS_SYNC                "Sync"
-    IDS_ENABLED             "Enabled"
-    IDS_ADVANCED_CALENDAR   "Advanced Calendar Settings"
-    IDS_DATA_FORMAT         "&Data format:"
+    IDS_ENABLED             "Aktiviert"
+    IDS_ADVANCED_CALENDAR   "Erweiterte Kalendereinstellungen"
+    IDS_DATA_FORMAT         "&Datenformat:"
     IDS_USE_SIF             "SIF"
     IDS_USE_VCARD           "vCard"
-    IDS_ENCRYPTION          "Encryption"
-    IDS_ADVANCED_CONTACTS   "Advanced Contact Settings"
-    IDS_FULLSYNC_PERFORM    "Perform full sync for:"
-    IDS_CONTACTS            "Contact&s"
-    IDS_CALENDAR            "Cal&endar"
-    IDS_NOTES               "&Notes"
-    IDS_TASKS               "&Tasks"
+    IDS_ENCRYPTION          "Verschlüsselung"
+    IDS_ADVANCED_CONTACTS   "Erweiterte Kontakteinstellungen"
+    IDS_FULLSYNC_PERFORM    "Vollständige Synchronisation ausführen für:"
+    IDS_CONTACTS            "Kontakt&e"
+    IDS_CALENDAR            "Kal&ender"
+    IDS_NOTES               "&Notizen"
+    IDS_TASKS               "&Aufgaben"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_SYNCHRONIZE         "Synchronize"
-    IDS_LOG_LEVEL           "Log Level"
-    IDS_LOG_NONE            "Error"
+    IDS_SYNCHRONIZE         "Synchronisieren"
+    IDS_LOG_LEVEL           "Log-Level"
+    IDS_LOG_NONE            "Fehler"
     IDS_LOG_INFO            "Info"
     IDS_LOG_DEBUG           "Debug"
-    IDS_ADVANCED_NOTES      "Notes Advanced Settings"
-    IDS_ADVANCED_TASKS      "Advanced Tasks Settings "
+    IDS_ADVANCED_NOTES      "Erweiterte Notizeinstellungen"
+    IDS_ADVANCED_TASKS      "Erweiterte Aufgabeneinstellungen"
     IDS_ERROR_CANNOT_CHANGE_SETTINGS 
-                            "Sync is in progress. Cannot change the settings."
-    IDS_ERROR_SET_URL       "Please set the url"
-    IDS_ERROR_SET_USERNAME  "Please set the user name"
-    IDS_ERROR_SET_PASSWORD  "Please set the password"
-    IDS_ERROR_PROXY_USERNAME "Please set the proxy user name"
-    IDS_ERROR_PROXY_PASSWORD "Please set the proxy password"
-    IDS_ERROR_SET_REMOTE_NAME "Please set the remote name"
-    IDS_SCHEDULER_EVERY     "Every"
-    IDS_MINUTES             "minutes"
+                            "Daten werden gerade synchronisiert. Es können keine Einstellungen geändert werden."
+    IDS_ERROR_SET_URL       "Bitte URL eingeben"
+    IDS_ERROR_SET_USERNAME  "Bitte Benutzername eingeben"
+    IDS_ERROR_SET_PASSWORD  "Bitte Passwort eingeben"
+    IDS_ERROR_PROXY_USERNAME "Bitte Proxy-Benutzername eingeben"
+    IDS_ERROR_PROXY_PASSWORD "Bitte Proxy-Passwort eingeben"
+    IDS_ERROR_SET_REMOTE_NAME "Bitte Remote-Name eingeben"
+    IDS_SCHEDULER_EVERY     "Alle"
+    IDS_MINUTES             "Minuten"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_CODE_NOSOURCE_10    "No sources to sync"
-    IDS_CODE_SYNC_STOPPED   "Synchronization terminated by the user"
-    IDS_CODE_AUTH_EXPIRED_402 "Error: account expired."
-    IDS_CODE_NOTFOUND_404   "Can't recognize remote name. Correct remote name in Sync Options."
+    IDS_CODE_NOSOURCE_10    "Keine Quelle für die Synchronisation gefunden"
+    IDS_CODE_SYNC_STOPPED   "Synchronisation durch den Benutzer beendet"
+    IDS_CODE_AUTH_EXPIRED_402 "Fehler: Benutzerkonto nicht mehr aktiv."
+    IDS_CODE_NOTFOUND_404   "Remote-Name kann nicht erkannt werden. Bitte korrekten Remote-Namen unter Synchronisationsoptionen angeben."
     IDS_CODE_INVALID_CREDENTIALS_401 
-                            "Can't access your account. Correct your server location, username or password."
+                            "Es kann keine Verbindung hergestellt werden. Bitte URL, Benutzername oder Passwort überprüfen."
     IDS_CODE_ERROR_CONNECT_2001 
-                            "Can't access the server. Check your network connectivity and your account's server location."
+                            "Es kann keine Verbindung zum Server hergestellt werden. Bitte Netzwerkverbindung und URL des Benutzerkontos überprüfen."
     IDS_CODE_ERROR_READING_CONTENT_2002 
-                            "Error reading content. Server unavailable."
+                            "Fehler beim Lesen der Dateninhalte. Server ist nicht verfügbar."
     IDS_CODE_SERVER_NOT_FOUND_2003 
-                            "Error: server not found. Check server uri."
+                            "Fehler: Server konnte nicht gefunden werden. Bitte URL überprüfen."
     IDS_CODE_INTERNET_CONNECTION_MISSING_2005 
-                            "Error: Cannot establish an internet connection."
-    IDS_CODE_NETWORK_ERROR_2007 "Sync not completed. Network error."
-    IDS_ERROR_SYNC_NOT_COMPLETED "Sync not completed. See log for details"
-    IDS_TEXT_APPOINTMENTS   "appointments"
-    IDS_TEXT_CONTACTS       "contacts"
-    IDS_TEXT_NOTES          "notes"
-    IDS_TEXT_TASKS          "tasks"
-    IDS_TEXT_SENDING        "Sending"
+                            "Fehler: Es konnte keine Internet-Verbindung hergestellt werden."
+    IDS_CODE_NETWORK_ERROR_2007 "Synchronisation konnte aufgrund eines Netzwerkfehlers nicht abgeschlossen werden."
+    IDS_ERROR_SYNC_NOT_COMPLETED "Synchronisation konnte nicht abgeschlossen werden. Details können der Log-Datei entnommen werden."
+    IDS_TEXT_APPOINTMENTS   "Termine"
+    IDS_TEXT_CONTACTS       "Kontakte"
+    IDS_TEXT_NOTES          "Notizen"
+    IDS_TEXT_TASKS          "Aufgaben"
+    IDS_TEXT_SENDING        "Übertragen"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_MAIN_CONTACTS       "Contacts"
-    IDS_MAIN_TASKS          "Tasks"
-    IDS_MAIN_NOTES          "Notes"
+    IDS_MAIN_CONTACTS       "Kontakte"
+    IDS_MAIN_TASKS          "Aufgaben"
+    IDS_MAIN_NOTES          "Notizen"
     IDS_ERROR_SET_CONNECTION 
-                            "You must enter your account information before attempting a sync."
-    IDS_ABOUT_TITLE         "About Funambol Outlook Sync Client"
-    IDS_MAIN_LAST_SYNC_FAILED "Last sync failed"
-    IDS_MAIN_LAST_SYNC_CANCELED "Last sync canceled"
+                            "Die Synchronisation kann nicht gestartet werden. Es müssen zuerst die Benutzerinformationen angegeben werden."
+    IDS_ABOUT_TITLE         "Über Funambol Outlook Sync Client"
+    IDS_MAIN_LAST_SYNC_FAILED "Letzte Synchronisation war nicht erfolgreich"
+    IDS_MAIN_LAST_SYNC_CANCELED "Letzte Synchronisation wurde abgebrochen"
     IDS_CODE_HTTP_NOT_FOUND_2060 
-                            "Error 404: resource not found. Please check your credentials."
-    IDS_UNEXPECTED_EXCEPTION "An unexpected exception has occurred!"
-    IDS_SYNCALL             "Sync All"
-    IDS_CODE_DROPPED_ITEMS  "Warning: some items were not correctly saved in Outlook."
-    IDS_TEXT_CHECKING       "Checking"
-    IDS_FINISHED_SENDING    "Waiting for Server data..."
+                            "Fehler 404: Server konnte nicht gefunden werden. Bitte Angaben überprüfen."
+    IDS_UNEXPECTED_EXCEPTION "Ein unerwarteter Fehler ist aufgetreten!"
+    IDS_SYNCALL             "alles synchronisieren"
+    IDS_CODE_DROPPED_ITEMS  "Warnung: Einige Einträge konnten nicht korrekt in Outlook übernommen werden."
+    IDS_TEXT_CHECKING       "Überprüfen"
+    IDS_FINISHED_SENDING    "Es werden auf die Severdaten gewartet..."
     IDS_GUIDE_LINK          "http://funambol.com/docs/v71/funambol-outlook-sync-client-v7.1-user-guide.pdf"
-    IDS_ERR_SERVER_TIMOUT   "Server returned timeout error: current session has expired.\nPlease retry later."
-    IDS_CODE_SERVER_BUSY    "Server is busy, please retry later."
+    IDS_ERR_SERVER_TIMOUT   "Server Time-out: die aktuelle Session wurde abgebrochen.\nBitte versuchen Sie es später."
+    IDS_CODE_SERVER_BUSY    "Server ist überlastet. Bitte versuchen Sie es später."
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_TEXT_RECEIVING      "Receiving"
-    IDS_TEXT_SYNC_ENDED     "Sync ended"
-    IDS_MAINBUT_SYNCHRONIZE "&Synchronize"
-    IDS_MAINBUT_CANCEL      "&Cancel"
-    IDS_TEXT_STARTING_SYNC  "Starting sync..."
-    IDS_TEXT_SYNC_CANCELED  "Sync canceled"
-    IDS_LEFTVIEW_ACCOUNT    "Account"
-    IDS_LEFTVIEW_SYNCHRONIZATION "Synchronization"
-    IDS_HOURS               "hours"
-    IDS_SCHEDULER_PLEASE_SET_VALUE "Please set the scheduled value."
-    IDS_SCHEDULER_MIN_VALUE_IS "The minimum value is"
-    IDS_SCHEDULER_MAX_VALUE_IS "The maximum value is"
-    IDS_SCHEDULER_CANNOT_SCHEDULE "Cannot schedule"
-    IDS_LEFTVIEW_CONTACTS   "Contacts"
-    IDS_LEFTVIEW_CALENDAR   "Calendar"
-    IDS_LEFTVIEW_TASKS      "Tasks"
+    IDS_TEXT_RECEIVING      "Empfangen"
+    IDS_TEXT_SYNC_ENDED     "Synchronisation beendet"
+    IDS_MAINBUT_SYNCHRONIZE "&Synchronisieren"
+    IDS_MAINBUT_CANCEL      "&Abbrechen"
+    IDS_TEXT_STARTING_SYNC  "Synchronisation starten..."
+    IDS_TEXT_SYNC_CANCELED  "Synchronisation abbrechen"
+    IDS_LEFTVIEW_ACCOUNT    "Benutzerkonto"
+    IDS_LEFTVIEW_SYNCHRONIZATION "Synchronisation"
+    IDS_HOURS               "Stunden"
+    IDS_SCHEDULER_PLEASE_SET_VALUE "Bitte den geplanten Wert eingeben."
+    IDS_SCHEDULER_MIN_VALUE_IS "Der minimale Wert ist"
+    IDS_SCHEDULER_MAX_VALUE_IS "Der maximale Wert ist"
+    IDS_SCHEDULER_CANNOT_SCHEDULE "Es kann nicht geplant werden"
+    IDS_LEFTVIEW_CONTACTS   "Kontakte"
+    IDS_LEFTVIEW_CALENDAR   "Kalender"
+    IDS_LEFTVIEW_TASKS      "Aufgaben"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_LEFTVIEW_NOTES      "Notes"
-    IDS_SCHEDULER           "Scheduler"
+    IDS_LEFTVIEW_NOTES      "Notizen"
+    IDS_SCHEDULER           "Planung"
     IDS_LEFTVIEW_LOG        "Log"
-    IDS_DAYS                "days"
+    IDS_DAYS                "Tage"
     IDS_MSG_CLOSE_SYNC_IN_PROGRESS 
-                            "Synchronization in progress. Please wait or cancel it."
+                            "Synchronisation wird gerade ausgeführt. Bitte warten Sie oder brechen Sie die Synchronisation ab."
     IDS_ABOUT_COPYRIGHT     "Copyright © 2003 - 2009 Funambol, Inc.\nAll rights reserved."
     IDS_TEXT_SYNC_ALREADY_RUNNING 
-                            "Sync already in progress.\nPlease wait until the sync is finished."
+                            "Synchronisation wird gerade ausgeführt.\nBitte warten Sie, bis die Synchronisation beendet ist."
     IDS_SYNC_ABORTED_AVOID_FULLSYNC 
-                            "Sync aborted by the user to avoid full-sync"
-    IDS_TEXT_SYNC_IN_PROGRESS "A scheduled sync is currently in progress."
+                            "Die Synchronisation wurde vom Benutzer abgebrochen, um eine vollständige Synchronisation zu verhindern."
+    IDS_TEXT_SYNC_IN_PROGRESS "Eine geplante Synchronisation wird gerade ausgeführt."
     IDS_DETAILS             "&Details..."
-    IDS_SYNC_SETTINGS       "Synchronization Settings"
-    IDS_APPLY               "&Apply"
+    IDS_SYNC_SETTINGS       "Synchronisationseinstellungen"
+    IDS_APPLY               "&Akzeptieren"
     IDS_ERROR_SYNC_TERMINATED 
-                            "Sync process could not be terminated correctly.\nOutlook Sync Client will now be closed."
-    IDS_TEXT_CANCELING_SYNC "Canceling sync, please wait..."
-    IDS_MAIN_PRESS_TO_SYNC  "Sync All"
-    IDS_MAIN_PRESS_TO_CANCEL "Cancel Sync"
+                            "Die Synchronisation konnte nicht erfolgreich abgeschlossen werden.\n Der Outlook Sync Client wird beendet."
+    IDS_TEXT_CANCELING_SYNC "Synchronisation wird abgebrochen. Bitte warten..."
+    IDS_MAIN_PRESS_TO_SYNC  "Alles synchronisieren"
+    IDS_MAIN_PRESS_TO_CANCEL "Synchronisation abbrechen"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_DONE                "Done"
+    IDS_DONE                "Erledigt"
     IDS_SERVER              "Server"
-    IDS_PROXY_SETTINGS      "&Proxy Settings…"
-    IDS_PROXY_SETTINGS2     "Proxy Settings"
-    IDS_ITEMS               "Items"
-    IDS_SYNC_SYNCHRONIZE_EVERY "S&ynchronize every"
-    IDS_SECURITY            "Security"
-    IDS_SYNC_ENABLE_ENCRYPTION "Enable enc&ryption"
-    IDS_CONTACTS_DETAILS    "Contacts Details"
-    IDS_SYNC_DIRECTION      "&Sync Direction"
-    IDS_CONTACTS_FOLDER     "Contacts &Folder"
-    IDS_CURRENT             "Current:"
-    IDS_ADVANCED            "Ad&vanced"
-    IDS_CALENDAR_DETAILS    "Calendar Details"
-    IDS_CALENDAR_FOLDER     "Calendar &Folder"
+    IDS_PROXY_SETTINGS      "&Proxy-Einstellungen…"
+    IDS_PROXY_SETTINGS2     "Proxy-Einstellungen"
+    IDS_ITEMS               "Einträge"
+    IDS_SYNC_SYNCHRONIZE_EVERY "S&ynchronisation alle"
+    IDS_SECURITY            "Sicherheit"
+    IDS_SYNC_ENABLE_ENCRYPTION "Ver&schlüsselung aktivieren"
+    IDS_CONTACTS_DETAILS    "Kontakt-Details"
+    IDS_SYNC_DIRECTION      "&Synchronisation"
+    IDS_CONTACTS_FOLDER     "Kontakt-Ordner"
+    IDS_CURRENT             "Aktuell:"
+    IDS_ADVANCED            "Er&weitert"
+    IDS_CALENDAR_DETAILS    "Kalender-Details"
+    IDS_CALENDAR_FOLDER     "Kalender-Ordner"
     IDS_USE_VCAL            "vCalendar"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_TASKS_DETAILS       "Tasks Details"
-    IDS_TASKS_FOLDER        "Tasks &Folder"
-    IDS_NOTES_DETAILS       "Notes Details"
-    IDS_NOTES_FOLDER        "Notes &Folder"
+    IDS_TASKS_DETAILS       "Aufgaben-Details"
+    IDS_TASKS_FOLDER        "Aufgaben-Ordner"
+    IDS_NOTES_DETAILS       "Notizen-Details"
+    IDS_NOTES_FOLDER        "Notizen-Ordner"
     IDS_RECOVER_PERFORMS    "Recover performs a “full sync” operation to refresh all data in Outlook or on the server."
     IDS_DIRECTION           "&Direction"
-    IDS_FULLSYNC_SYNCTYPE2  "Replace all of the Outlook data with data from the server"
-    IDS_FULLSYNC_SYNCTYPE1  "Two-way"
-    IDS_FULLSYNC_SYNCTYPE3  "Replace all of the server data with data from Outlook"
-    IDS_RECOVER             "Recover"
+    IDS_FULLSYNC_SYNCTYPE2  "Sämtliche Daten von Microsoft Outlook mit Daten vom Server ersetzen."
+    IDS_FULLSYNC_SYNCTYPE1  "E"
+    IDS_FULLSYNC_SYNCTYPE3  "Sämtliche Daten vom Server mit Daten von Microsoft Outlook ersetzen."
+    IDS_RECOVER             "Wiederherstellen"
     IDS_LOGGING             "Logging"
-    IDS_NOT_SYNCHRONIZED    "Not synchronized"
+    IDS_NOT_SYNCHRONIZED    "Nicht synchronisiert"
     IDS_FUNAMBOL_LINK       "www.funambol.com"
-    IDS_SYNCHRONIZED        "Last sync:"
-    IDS_AT                  "at"
-    IDS_MAIN_CALENDAR       "Calendar"
+    IDS_SYNCHRONIZED        "Letzte Sychronisation:"
+    IDS_AT                  "um"
+    IDS_MAIN_CALENDAR       "Kalender"
 END
 
 STRINGTABLE 
 BEGIN
     IDS_CODE_SERVER_BUSY_SYNC 
-                            "The server is busy. It is possible that another sync is in progress for this account on another device. Please try later."
-    IDS_CODE_SERVER_ERROR_2052 "Sync not completed: Server failure."
+                            "Der Server ist überlastet. Es ist möglich, dass eine andere Synchronisation für dieses Benutzerkonto in Arbeit ist. Bitte versuchen Sie es später."
+    IDS_CODE_SERVER_ERROR_2052 "Synchroniation unterbrochen: Keine Verbindung zum Server."
     IDS_CODE_DROPPED_ITEMS_SERVER 
-                            "Warning: some items were not correctly transferred on Server."
-    IDS_CODE_NO_SOURCES     "No sources to sync.\nPlease select at least one source from Sync Options panel."
+                            "Warnung: Einige Daten konnten nicht korrekt zum Server transferiert werden."
+    IDS_CODE_NO_SOURCES     "Keine Quelle für die Synchronisation angegeben.\nBitte geben Sie mindestens eine Quelle an."
     IDS_HOUR                "hour"
-    IDS_CODE_FORBIDDEN_403  "Forbidden: please activate your account."
+    IDS_CODE_FORBIDDEN_403  "Nicht möglich: Bitte aktivieren Sie Ihr Benutzerkonto."
     IDS_USE_VTODO           "vCalendar"
     IDS_USE_VNOTE           "vNote"
     IDS_EVENT_FILTER        "Filter"
-    IDS_SYNC_PAST_EVENTS    "Sync Outlook appointments:"
-    IDS_DATE_FILTER_NONE    "Future only"
-    IDS_DATE_FILTER_LAST_WEEK "Since last week"
-    IDS_DATE_FILTER_LAST_MONTH "Since last month"
-    IDS_DATE_FILTER_LAST_3_MONTHS "Since last 3 months"
-    IDS_DATE_FILTER_LAST_6_MONTHS "Since last 6 months"
-    IDS_DATE_FILTER_ALL     "All"
+    IDS_SYNC_PAST_EVENTS    "Mit Outlook zu synchronisierende Termine:"
+    IDS_DATE_FILTER_NONE    "Nur zukünftige Termine"
+    IDS_DATE_FILTER_LAST_WEEK "Seit letzter Woche"
+    IDS_DATE_FILTER_LAST_MONTH "Seit letztem Monat"
+    IDS_DATE_FILTER_LAST_3_MONTHS "Seit den letzten 3 Monaten"
+    IDS_DATE_FILTER_LAST_6_MONTHS "Seit den letzten 6 Monaten"
+    IDS_DATE_FILTER_ALL     "Alle"
 END
 
 STRINGTABLE 
 BEGIN
-    IDS_DATE_FILTER_LAST_2_WEEKS "Since last 2 weeks"
+    IDS_DATE_FILTER_LAST_2_WEEKS "Seit der letzten 2 Wochen"
 END
 
 #endif    // German (Germany) resources
